app/views/merchants/invoices/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<center><h1>Little Esty Shop</h1></center><br>
+
+<h3><%= @merchant.name %></h3><br>
+
+<center><p>Invoice #<%= @invoice.id %></p></center>
+
+<p>Status: <%= @invoice.status %></p>
+<p>Created on: <%= @invoice.formatted_time %></p>
+<p>Total Revenue: <%= number_to_currency(@invoice.invoice_total_revenue / 100) %></p>
+
+<h3>Customer:</h3>
+<p><%= @invoice.customer.first_name + ' ' + @invoice.customer.last_name %></p><br>
+
+<p>Items on this Invoice:</p>
+<% @merchant.invoice_items_for_invoice(@invoice.id).each do |item| %>
+
+<div id="item_<%=item.id%>">
+  <p>Name: <%= Item.where('id = ?', item.item_id).first.name %></p>
```

Comment:
We should keep the business logic out of the views (i.e. the `Item.where...`).  Consider making a method within the Item model to follow good MVC practice